tasks/02_kubernetes_fundamentals/kubernetes_3/services.yaml

```diff
@@ -0,0 +1,79 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: ml-model-deployment
+  namespace: mlops
+  labels:
+    app: ml-model
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: ml-model
+  template:
+    metadata:
+      labels:
+        app: ml-model
+    spec:
+      containers:
+      - name: ml-model
+        image: mlops1
+        imagePullPolicy: Never
+        ports:
+        - containerPort: 5000
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: kube-model-api
+  namespace: mlops
+  labels:
+    app: model-api
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: model-api
+  template:
+    metadata:
+      labels:
+        app: model-api
+    spec:
+      containers:
+      - name: api-model-cont
+        image: mlops2
+        imagePullPolicy: Never
+        ports:
+        - containerPort: 5000
+---

```

Comment:
It's not a good practice to store all manifests in one file. I'd instead put deployment and service for one model per file.

Reply:
Done